src/other/Table.js

```diff
@@ -669,48 +695,47 @@
         }
     };
 
-    Table.prototype.applyHoverRowStyles = function(row){
-        var context = this;
-        row
-            .style("color",context.tbodyHoverRowFontColor())
-            .style("background-color",context.tbodyHoverRowBackgroundColor())
-        ;
-    };
-    Table.prototype.applySelectedRowStyles = function(row){
+    Table.prototype.getColumnFormatting = function(cellData,colIdx){
         var context = this;
-        row
-            .style("color",context.tbodySelectedRowFontColor())
-            .style("background-color",context.tbodySelectedRowBackgroundColor())
-        ;
-    };
-    Table.prototype.applyRowStyles = function(row, isFirstCol){
-        row
-            .style("color", isFirstCol ? this.tbodyFirstColFontColor() : this.tbodyFontColor())
-            .style("background-color", isFirstCol ? this.tbodyFirstColBackgroundColor() : this.tableZebraColor_exists() && this.data().indexOf(row.datum()) % 2 ? this.tbodyRowBackgroundColor() : this.tableZebraColor())
-        ;
-    };
-    Table.prototype.applyFirstColRowStyles = function(rows){
-        this.applyStyleToRows(rows,true);
+        if(typeof(cellData) === "string"){
+            var timeFormat = d3.time.format(context.columnPatterns()[colIdx]);
+            return timeFormat(new Date(cellData)) || cellData.trim();
+        } else if (typeof(cellData) === "number") {
+            var format = d3.format(context.columnPatterns()[colIdx]);
+            return format(cellData);
+        }
+        return cellData;
     };
-    Table.prototype.applyStyleToRows = function(rows,isFirstCol){
-        isFirstCol = typeof isFirstCol !== "undefined" ? isFirstCol : false;
+    Table.prototype.getColumnAlignment = function(cellData){
         var context = this;
-        rows.each(function () {
-                var tr = d3.select(this);
-                if (tr.classed("hover")) {
-                    context.applyHoverRowStyles(tr);
-                } else if (tr.classed("selected")) {
-                    context.applySelectedRowStyles(tr);
-                } else {
-                    context.applyRowStyles(tr,isFirstCol);
-                }
-            })
-        ;
+        switch(typeof(cellData)){
```

Comment:
more than likely this block right here will be replaced by calls to the new DB object